scripts/workers/code_upload_worker_utils/make_submission.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+submission_url="$EVALAI_API_SERVER/api/jobs/challenge/$CHALLENGE_PK/challenge_phase/$PHASE_PK/submission/$SUBMISSION_PK"
+submission_curl_base_request="curl --location --request PATCH '$submission_url' --header 'Authorization: Bearer $AUTH_TOKEN'"
+if [ -f "$SUBMISSION_PATH/submission.csv" ]
+then
+    file_path="submission.csv"
+    curl_request="$submission_curl_base_request -F \"submission_input_file=@$SUBMISSION_PATH/$file_path\""
+elif [ -f "$SUBMISSION_PATH/submission.json" ]
+then
+    file_path="submission.json"
+    curl_request="$submission_curl_base_request -F \"submission_input_file=@$SUBMISSION_PATH/$file_path\""
+else
+    exit 0
+fi
+echo "Submitting file to EvalAI..."
+eval $curl_request
+echo "\nFile submitted successfully"
+touch "$SUBMISSION_PATH/completed.txt"
```

Comment:
@savish28 instead of this you can use `echo $(date) > "$SUBMISSION_PATH/completed.txt"`. We don't need to create a file explicitly

Reply:
Updated.